src/Andromeda/AvaloniaApp.FSharp/InstallGame.fs

```diff
@@ -96,12 +137,12 @@ module InstallGame =
 
             let syncDispatch (dispatch: Dispatch<'msg>): Dispatch<'msg> =
                 match Dispatcher.UIThread.CheckAccess() with
-                | true -> fun msg -> Dispatcher.UIThread.Post(fun () -> dispatch msg)
-                | false -> dispatch
+                | true -> dispatch
+                | false -> fun msg -> Dispatcher.UIThread.Post(fun () -> dispatch msg)
 
             Program.mkProgram (init authentication) update view
             |> Program.withHost this
-            |> Program.withSyncDispatch syncDispatch
+            |> Program.withSyncDispatch (fun dispatch msg -> Dispatcher.UIThread.Post(fun () -> dispatch msg))
```

Comment:
I should revert that changes and check if everything works fine.
This stuff is a rest of some experiments to try and solve some nasty bugs I got while implementing this.